pupil-spa/nginx/default.conf

```diff
@@ -31,6 +31,20 @@ server {
     expires off;
   }
 
+  location /queue/ {
+    # resolver needed for the proxy_pass code below
+    resolver 8.8.8.8;
```

Comment:
There ought to be an Azure DNS server we can use.   E.g  from https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-name-resolution-for-vms-and-role-instances#azure-provided-name-resolution

| The Azure DNS IP address is 168.63.129.16. This is a static IP address and will not change.